dirty.html

```diff
@@ -1,49 +1,36 @@
 <html>  
   <head>
-    <h1>WDI London September 2013
+    <h1>WDI London September 2013</h1>
     <meta value="description" content="Web development immersive ">
   </head>
   <div class="body">
-    <div>
-        <div>
-        </div>
-        <div>
-          <div>Web Development Immersive <div>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</div></div>
+          <h1>Web Development Immersive </h1>
+          <h2>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</h2>
+          
           <div id="nav">
-            <div>
-              <div><link hef="#program">Program</link></div>
-              <div><link href="#students">Students</link></div>
-              <div><link hrf="#concepts">Concepts</link></div>
-              <div><link href="#instructors">Instructors</link></div>
-            </div>
-          </div>
-        </div>
-    </div>
-    <a name=program></a>
-    <div>
-      <div>
-        <div>
-          <div id="title" class="h3">The Program</div>
-          <a name="program"></link>
-          <div>              
-            <div>12 weeks of intensive web development training learning a range of programming techniques and languages.</d>
+            
+              <link href="#program">Program</link>
+              <link href="#students">Students</link>
+              <link href="#concepts">Concepts</link>
+              <link href="#instructors">Instructors</link>
           </div>
-        </div>
-        <div>
-          <div>
+    
+          <div id="title" 
```

Comment:
- open div missing a `>`
- also please consider adding the `id` to the `h3` bellow